package.json

```diff
@@ -24,6 +24,10 @@
   "types": "./dist/wavesurfer.d.ts",
   "browser": "./dist/wavesurfer.js",
   "exports": {
+    "./ts": {
+      "import": "./ts.ts",
+      "types": "./ts.ts"
```

Comment:
Did you mean `./src/ts.ts`?

Reply:
I put it in the root so it looks like `import { WaveSurfer, RegionsPlugin } from 'wavesurfer.js/ts'`